### PR TITLE
Bump the versions, new models support

### PR DESCRIPTION
**Pull request by JegernOUTT**

### CODE REVIEW

**Review by olegklimov — APPROVED** [review]

File: refact_webgui/webgui/static/tab-finetune.html [hunk 1]
```diff
@@ -191,7 +191,7 @@ <h5 class="modal-title" id="urlModalLabel">Launch Finetune</h5>
               <label class="form-label">Do you have a large codebase?</label>
               <div class="btn-group" role="group" aria-label="basic radio toggle button group">
                 <input type="radio" class="btn-check" name="trainable_embeddings" id="trainable_embeddings1" value="1" autocomplete="off">
-                <label class="btn btn-outline-primary" for="trainable_embeddings1">Yes, train embeddings</label>
+                <label class="btn btn-outline-primary" for="trainable_embeddings1">Yes, train embeddings (unstable with vllm)</label>
```
Comment (olegklimov) [inline refact_webgui/webgui/static/tab-finetune.html:194]: if vllm is the new default, then disable this completely, don't say it's unstable?

Comment (JegernOUTT) [reply refact_webgui/webgui/static/tab-finetune.html:194]: going to be changed in the next version when vllm actually become a new default provider

File: self_hosting_machinery/inference/inference_hf.py [hunk 2]
```diff
@@ -149,38 +149,32 @@ def __init__(self,
 
         self._device = "cuda:0"
         token = huggingface_hub_token()
-        for local_files_only in [True, False]:
```
Comment (mitya52) [inline self_hosting_machinery/inference/inference_hf.py:152]: what's the reason of this change?
how we should handle the case when we have no connection to internet?

File: refact_known_models/huggingface.py [hunk 3]
```diff
@@ -113,7 +116,37 @@
...
+        "backend": "transformers",
+        "model_path": "meta-llama/Llama-3.2-1B-Instruct",
+        "model_class_kwargs": {
+            "torch_dtype": "bf16",
+        },
+        "required_memory_mb": 8000,
+        "T": 16384,  # in fact this model can handle 128K context
+        "filter_caps": ["completion", "chat"],
     },
     "deepseek-coder-v2/16b/instruct": {
```
Comment (mitya52) [inline refact_known_models/huggingface.py:151]: we need to remove this model
no support for it in refact-lsp

File: Dockerfile.base [hunk 4]
```diff
@@ -1,7 +1,7 @@
-FROM nvidia/cuda:11.8.0-cudnn8-devel-ubuntu22.04
+FROM nvidia/cuda:12.4.1-cudnn-devel-ubuntu22.04
 
 ENV INSTALL_OPTIONAL=TRUE
-ENV MAX_JOBS=8
+ENV MAX_JOBS=32
```
Comment (mitya52) [inline Dockerfile.base:4]: we're building this image with gh actions
so 32 cores may be not available

**Review by mitya52 — CHANGES_REQUESTED** [review]

File: self_hosting_machinery/inference/inference_hf.py (hunk 2 above)
Comment (JegernOUTT) [reply self_hosting_machinery/inference/inference_hf.py:152]: It's handled automatically, there is no need for this code

File: refact_known_models/huggingface.py (hunk 3 above)
Comment (JegernOUTT) [reply refact_known_models/huggingface.py:151]: ok

File: Dockerfile.base (hunk 4 above)
Comment (JegernOUTT) [reply Dockerfile.base:4]: ok

**Review by mitya52 — APPROVED** [review]
